Changelog.md

```diff
@@ -16,20 +17,22 @@ Compiler Features:
 
 
 Bugfixes:
+ * ABI: Include events in the ABI that are emitted by a contract but defined outside of it.
  * Antlr Grammar: Fix discrepancy with the parser, which allowed octal numbers.
  * Antlr Grammar: Fix of a discrepancy with the parser, which allowed numbers followed by an identifier with no whitespace.
  * Antlr Grammar: Stricter rules for function definitions. The grammar will no longer accept as valid free functions having specifiers which are exclusive to contract functions.
  * SMTChecker: Fix false positives in ternary operators that contain verification targets in its branches, directly or indirectly.
 
 
 AST Changes:
- * AST: add the ``internalFunctionID`` field to the AST nodes of functions that may be called via the internal dispatch. These IDs are always generated, but they are only used in via-IR code generation.
+ * AST: Add the ``internalFunctionID`` field to the AST nodes of functions that may be called via the internal dispatch. These IDs are always generated, but they are only used in via-IR code generation.
```

Comment:
Why is this line showing up in the diff by the way? Not sure if I'm blind, but I don't see any change in it...

Reply:
Haha, lol - but that's also a bug in github - it usually highlights the actual change, here it doesn't :-)